package.json

```diff
@@ -46,12 +46,12 @@
     "homepage": "https://github.com/textkernel/oneui#readme",
```

Comment:
![Screenshot 2019-08-06 at 11 45 35](https://user-images.githubusercontent.com/19436304/62529842-bb84be00-b83f-11e9-80f7-6b28ff01f1de.png)
Missing pin icon on the left of the input

Reply:
added